src/main/java/ch/njol/skript/hooks/regions/conditions/CondRegionContains.java

```diff
@@ -77,25 +58,16 @@ public boolean init(final Expression<?>[] exprs, final int matchedPattern, final
 		setNegated(matchedPattern >= 2);
 		return true;
 	}
-	
+
 	@Override
 	public boolean check(final Event e) {
-		return regions.check(e, new Checker<Region>() {
-			@Override
-			public boolean check(final Region r) {
-				return locs.check(e, new Checker<Location>() {
-					@Override
-					public boolean check(final Location l) {
-						return r.contains(l);
-					}
-				}, isNegated());
-			}
-		});
+		return regions.check(e,
+			r -> locs.check(e, r::contains, isNegated()));
```

Comment:
I know it's sort of out of scope, but this results in `locs` expression being evaluated multiple times.

Reply:
do i fix it by reversing the expressions so loc checks regions?

Reply:
If you want to fix the occurrences in this PR (you don't necessarily need to), you can do the normal getArray outside of the checker, and then in the checker, there is a static `check` method on SimpleExpression you can use

Reply:
ill create a separate pr for it